frontend/express/public/javascripts/countly/countly.vue.components.js

```diff
@@ -372,4 +373,2886 @@
+                            </ul>\
+                            <div class="cly-vue-tabs-container">\
+                                <slot/>\
+                            </div>\
+                        </div>'
+        }
+    ));
+
+    Vue.component("cly-content", countlyBaseComponent.extend(
+        // @vue/component
+        {
+            inheritAttrs: false,
+            mixins: [
+                _mixins.i18n
+            ],
+            props: {
+                name: { type: String, default: null},
+                id: { type: String, default: null },
+                alwaysMounted: { type: Boolean, default: true },
+                alwaysActive: { type: Boolean, default: false },
+                role: { type: String, default: "default" }
+            },
+            data: function() {
+                return {
+                    isContent: true
+                };
+            },
+            computed: {
+                isActive: function() {
+                    return this.alwaysActive || (this.role === "default" && this.$parent.activeContentId === this.id);
+                },
+                tName: function() {
+                    return this.name;
+                },
+                tId: function() {
+                    return this.id;
+                },
+                elementId: function() {
+                    return this.componentId + "-" + this.id;
+                }
+            },
+            template: '<div class="cly-vue-content" :id="elementId" v-if="isActive || alwaysMounted">\
+                            <div v-show="isActive"><slot/></div>\
+                        </div>'
+        }
+    ));
+
+    Vue.component("cly-panel", countlyBaseComponent.extend(
+        // @vue/component
+        {
+            props: {
+                title: { type: String, required: false },
+                dateSelector: { type: Boolean, required: false, default: true },
+                hasLeftBottom: { type: Boolean, required: false, default: false },
+                onlyHead: { type: Boolean, required: false, default: false }
+            },
+            template: '<div class="cly-vue-panel widget">\
+                            <div class="widget-header">\
+                                <div class="left">\
+                                    <div style="margin-left: 3px;">\
+                                        <slot name="left-top">\
+                                            <div class="title" :class="{small: hasLeftBottom}">{{title}}</div>\
+                                        </slot>\
+                                        <div v-if="hasLeftBottom">\
+                                            <slot name="left-bottom"></slot>\
+                                        </div>\
+                                    </div>\
+                                </div>\
+                                <div class="right">\
+                                    <slot name="right-top">\
```

Comment:
Codacy found an issue: [Bad escaping of EOL. Use option multistr if needed.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=6339769)